Run the copy also in check mode

martbhell opened the pull request:
- Without it the next task (yum install of said rpm) will fail if the file is not there

martbhell committed:
Run the copy also in check mode

 - Without it the next task will fail if the file is not there

oscarkraemer reviewed (approved):
I would have skipped the yum install instead of the copy, but there are multiple ways to skin a cat. Travis test is failing on the centos-devel, if you want to fix that feel free. If you do not want to fix that I'm okay with that as well, feel free to merge the PR.

martbhell committed:
Allow specifying workers for gearman

And test changing these values

martbhell committed:
Fix testing by installing an older version of pip

HTTPSConnectionPool(host='pypi.org', port=443): Max retries exceeded
with url: /simple/pyrax/ (Caused by SSLError(SSLError("bad handshake:
Error([('SSL routines', 'ssl3_get_server_certificate', 'certificate
verify failed')],)",),)) - skipping
